.github/workflows/CI-samples.yml

```diff
@@ -0,0 +1,23 @@
+name: CI-docs
```

Comment:
Sample tested on all platform via https://github.com/libuv/libuv/pull/3592

Reply:
Looks like they are very slightly different in that this tests the Makefile. But I would be inclined to just delete the Makefile, now that we have a CMakeFile.